deltachat-ios/Controller/ChatViewController.swift

```diff
@@ -381,6 +390,17 @@ class ChatViewController: MessagesViewController {
         }
     }
 
+    func scrollToMessage(id: Int, animated: Bool) {
+        for (index, msg) in messageList.enumerated() {
+            if msg.id == id {
+                let indexPath = IndexPath(row: 0, section: index)
+                messagesCollectionView.scrollToItem(at: indexPath, at: .top, animated: animated)
+                return
+            }
+        }
+        safe_fatalError("message could not be found")
```

Comment:
this is too harsh. On my first try, the app already crashed. The reason was that the message was not within this first ones that are loaded, but already 'hidden'. I would have to pull to refresh in the chat view to have the searched message in the list.

Reply:
Ok, this will not crash in production but still it's annoying a bit.

Reply:
doesn't that always happen when clicking on a search result that is not in the last 100 or so messages?

Reply:
Ok, I probably got this wrong. I didn't know/considered, that messages are incrementally fetched